lib/plain/plain-client.ts

```diff
@@ -65,7 +65,9 @@ export const createPlainClient = (params: ClientParams, defaults?: DefaultParams
     user: {
       getManyForSpace: wrap(wrapParams, endpoints.user.getManyForSpace),
       getForSpace: wrap(wrapParams, endpoints.user.getForSpace),
-      getCurrent: wrapHttp(http, endpoints.user.getCurrent),
+      getCurrent: (...args: RestParamsType<typeof endpoints.user.getCurrent>) => {
```

Comment:
For my own education, is there a reason we have to diverge from the `getForSpace`, etc from above?

Reply:
because wrap() applies default params that we don't want.
```
{
   spaceId,
   environmentId,
   organizationId
}
```